manyfews/calculations/models.py

```diff
@@ -37,3 +37,18 @@ class InitialCondition(models.Model):
     storage_level = models.FloatField()
     slow_flow_rate = models.FloatField()
     fast_flow_rate = models.FloatField()
+
+
+class RainAndEvapotranspiration(models.Model):
```

Comment:
Could you add a comment or Docstring explaining what this is for? (I don't currently understand how this is used.)

Reply:
Do I still need to add comments to the two tables?